salt/transport/tcp.py

```diff
@@ -631,26 +839,25 @@ def getstream(self, **kwargs):
                     exc,
                     self.backoff,
                 )
-                yield tornado.gen.sleep(self.backoff)
-        raise tornado.gen.Return(stream)
+                await asyncio.sleep(self.backoff)
+        return stream
 
-    @tornado.gen.coroutine
-    def connect(self):
+    async def connect(self):
         if self._stream is None:
-            self._stream = yield self.getstream()
+            self._stream = await self.getstream()
             if self._stream:
                 if not self._stream_return_running:
-                    self.io_loop.spawn_callback(self._stream_return)
+                    self.task = asyncio.create_task(self._stream_return())
+                    # self.io_loop.spawn_callback(self._stream_return)
```

Comment:
```suggestion
```

Reply:
This is in the MessageClient class which has been deprecated. We can safely ignore it.